src/main.nqp

```diff
@@ -32,7 +32,6 @@ my $install-dir := $execname eq ''
     !! nqp::substr($execname, 0, nqp::rindex($execname, $sep, nqp::rindex($execname, $sep) - 1));
 
 my $rakudo-home := $comp.config<static_rakudo_home>
-    // nqp::getenvhash()<PERL6_HOME>
```

Comment:
Does this need a deprecation cycle?
I'd suspect the *_HOME variables have been used rarely, if at all, since their introduction. But it's impossible to tell for sure.

Reply:
Sure it does.